packages/auth/__tests__/auth0.test.ts

```diff
@@ -0,0 +1,173 @@
+      '.auth0-lock-input-email input',
+    );
+    const passwordInput = await page.waitForSelector(
+      '.auth0-lock-input-password input',
+    );
+
+    await emailInput.focus();
+    await emailInput.type(AUTH_EMAIL);
+    await passwordInput.focus();
+    await passwordInput.type(AUTH_PASSWORD);
+
+    await page.click('button.auth0-lock-submit');
+
+    await page.waitForNavigation({
+      waitUntil: 'networkidle2',
+    });
+
+    expect(page.url()).toMatch(SERVER_URL);
+
+    await page.evaluate(
+      createGlobalAuth,
+      AUTH_PROFILE_ID,
+      AUTH0_DOMAIN,
+      AUTH0_CLIENT_ID,
+      SERVER_URL,
+    );
+
+    const authorizedData = await page.evaluate(() => {
+      return auth.getAuthorizedData();
+    });
+
+    expect(authorizedData).toMatchObject({
+      accessToken: expect.any(String),
+      idToken: expect.any(String),
+      expiresIn: expect.any(Number),
+      idTokenPayload: {
+        email: expect.any(String),
+        email_verified: expect.any(Boolean),
+      },
+    });
+  });
+
+  it('returns current info about authorized user', async () => {
+    const currentUserData = await page.evaluate(() => {
+      return auth.currentUser();
+    });
+
+    expect(currentUserData).toMatchObject({
+      idToken: expect.any(String),
+      email: expect.any(String),
+      emailVerified: expect.any(Boolean),
+      accessToken: expect.any(String),
+      expiresIn: expect.any(Number),
+      idTokenPayload: expect.any(Object),
+    });
+  });
+
+  it('refreshes idToken', async () => {
+    const refreshedAuthorizedData = await page.evaluate(() => {
+      return auth.refresh();
+    });
+
+    expect(refreshedAuthorizedData).toMatchObject({
+      accessToken: expect.any(String),
+      idToken: expect.any(String),
+      expiresIn: expect.any(Number),
+      idTokenPayload: {
+        email: expect.any(String),
+        email_verified: expect.any(Boolean),
+      },
+    });
+
+    const currentUserData = await page.evaluate(() => {
+      return auth.currentUser();
+    });
+
+    expect(currentUserData).toMatchObject({
+      idToken: refreshedAuthorizedData.idToken,
+      email: refreshedAuthorizedData.idTokenPayload.email,
+      emailVerified: refreshedAuthorizedData.idTokenPayload.email_verified,
+      accessToken: refreshedAuthorizedData.accessToken,
+      expiresIn: refreshedAuthorizedData.expiresIn,
+      idTokenPayload: refreshedAuthorizedData.idTokenPayload,
+    });
+  });
+
+  it('signs out', async () => {
+    await page.evaluate(() => {
+      auth.signOut();
+    });
+
+    await page.waitForNavigation();
+
+    expect(page.url()).toMatch(SERVER_URL);
+
+    await page.evaluate(
+      createGlobalAuth,
+      AUTH_PROFILE_ID,
+      AUTH0_DOMAIN,
+      AUTH0_CLIENT_ID,
+      SERVER_URL,
+    );
+
+    const currentUserData = await page.evaluate(() => {
+      return auth.currentUser();
+    });
+
+    expect(currentUserData).toEqual({});
+  });
```

Comment:
I think we're missing the "auth.signIn()". This would only work, obviously, for email/password.

```
/* Let's user sign in */
Auth.signIn(username: String!, password: String)

/* EXAMPLE */
import Auth from 'src/auth.js'

/* Login User */
const {
  success,
  idToken,
  strategy,
  confirmed
} = await Auth.signIn("my@email.com", "awesomePassword!")
```